pyparsing/core.py

```diff
@@ -4174,17 +4188,14 @@ def _setResultsName(self, name, listAllMatches=False):
                 not in e.suppress_warnings_
                 for e in self.exprs
             ):
-                warnings.warn(
```

Comment:
Accepted. +1 for f-strings, but please move separating spaces to beginning of continuation lines instead of end of lines.